util/runMasonCI.bash

```diff
@@ -37,10 +37,10 @@ checkPackage () {
   basename "$FILE"
   f="$(basename -- "$FILE")"
   source="$(grep source "$f" | cut -d= -f2)"
-
+  echo $source
```

Comment:
Would it be useful to print out a label or anything for what's about to be echo'd, here and below?

Reply:
these are all just temporary measures to figure out what is wrong - I don't expect them to persist, but if they do, then yes, some label will be appropriate to add